cookbooks/aws-parallelcluster-config/templates/default/init/fetch_and_run.erb

```diff
@@ -21,22 +21,23 @@ function error_exit () {
   script=`basename $0`
   echo "parallelcluster: ${script} - $1"
   logger -t parallelcluster "${script} - $1"
+  echo "$1. If --rollback-on-failure was set to false, check /var/log/cfn-init.log and /var/log/cloud-init-output.log for more details." > /var/log/parallelcluster/bootstrap_error_msg
```

Comment:
I think I would not add `. If --rollback-on-failure was set to false, check /var/log/cfn-init.log and /var/log/cloud-init-output.log for more details.` to the error message file - we should do that when we call cfn-signal.

Reply:
Yeah I wanted to do that as well, but I realized the logs we want the customers to check might differ according to the sources: whether it's from BYOS recipes, other chef recipes, or custom scripts. For example, for chef recipes, we would suggest them also check `chef-client.log`.

Reply:
👍